lite/backends/arm/math/conv5x5s2_depthwise_fp32.cc

```diff
@@ -13,733 +13,1801 @@
+  auto act_type = act_param.active_type;
+  float tmp = act_param.Relu_clipped_coef;
+  float ss = act_param.Leaky_relu_alpha;
+  float vsix[4] = {tmp, tmp, tmp, tmp};
+  float vscale[4] = {ss, ss, ss, ss};
+  if (has_active) {
+    switch (act_type) {
+      case lite_api::ActivationType::kRelu:
+        conv_depthwise_5x5s2_bias_relu(dout,
+                                       din,
+                                       weights,
+                                       bias,
+                                       flag_bias,
+                                       num,
+                                       chin,
+                                       hin,
+                                       win,
+                                       hout,
+                                       wout,
+                                       pad_top,
+                                       pad_bottom,
+                                       pad_left,
+                                       pad_right,
+                                       ctx);
+        break;
+      case lite_api::ActivationType::kRelu6:
+        conv_depthwise_5x5s2_bias_relu6(dout,
+                                        din,
+                                        weights,
+                                        bias,
+                                        vsix,
+                                        flag_bias,
+                                        num,
+                                        chin,
+                                        hin,
+                                        win,
+                                        hout,
+                                        wout,
+                                        pad_top,
+                                        pad_bottom,
+                                        pad_left,
+                                        pad_right,
+                                        ctx);
+        break;
+      case lite_api::ActivationType::kLeakyRelu:
+        conv_depthwise_5x5s2_bias_leakyRelu(dout,
+                                            din,
+                                            weights,
+                                            bias,
+                                            vscale,
+                                            flag_bias,
+                                            num,
+                                            chin,
+                                            hin,
+                                            win,
+                                            hout,
+                                            wout,
+                                            pad_top,
+                                            pad_bottom,
+                                            pad_left,
+                                            pad_right,
+                                            ctx);
+        break;
+      default:
+        LOG(FATAL) << "this act_type: " << static_cast<int>(act_type)
```

Comment:
```suggestion
        LOG(FATAL) << "this act_type: " << ActivationTypeToStr(static_cast<int>(act_type))
```

可以考虑转换成对应的名字，实现在paddle_api.h里。非必须